packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -13,13 +13,13 @@ A "floating" Promise is one that is created without any code set up to handle an
 Floating Promises can cause several issues, such as improperly sequenced operations, ignored Promise rejections, and more.
 
 This rule reports when a Promise is created and not properly handled.
-Valid ways of handling a Promise-valued statement include:
+Valid ways of silencing a Promise-valued statement include:
```

Comment:
What do you think of something more along these lines?

```suggestion
This rule will report Promise-valued statements that are not treated in one of the following ways:
```

The idea I'm getting at is to just say what's reported or not reported, and dodge unpacking every scenario that does or doesn't lead to unhandled rejection, and why the rule cannot report in 1:1 correspondence with those scenarios.

Reply:
Yeah, I chose to just use "silencing" because of the issue I've opened before this PR.
But yeah, I like this.